### PR TITLE
[SNAKE-3] Added .gitignore file

### PR DESCRIPTION
**Pull request by teclamat**
Please review

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.vscode
```
Comment (teclamat) [inline .gitignore:1]: Looks good

**Review by BorediXxX — COMMENTED** [review]
yeah

**Review by teclamat — COMMENTED** [review]
There was really nothing to review but whatever

File: .gitignore (hunk 1 above)
Comment (km44192) [inline .gitignore:1]: Everything looks OK

**Review by km44192 — APPROVED** [review]

**Pull request merged**